src/photos/photoData.ts

```diff
@@ -0,0 +1,25 @@
+export interface TrimmedPhotoData {
+    id: number;
+    img_url: string;
+    sol: number;
+    earth_date: string;
+}
+
+
+export function trimPhotoData(photos: any[]): TrimmedPhotoData[] {
+    const trimmedPhotos: TrimmedPhotoData[] = [];
+
+    while(photos.length > 0) {
+        const photoData = photos.pop();
+        const trimmedPhotoData: TrimmedPhotoData = {
+            id:photoData.id,
+            img_url:photoData.img_src,
+            sol:photoData.sol,
+            earth_date:photoData.earth_date
```

Comment:
Probably should have a check that this data exists in the response from the NASA api, I just trusted that it does.